src/components/SimplePagination/_simple-pagination.scss

```diff
@@ -0,0 +1,28 @@
+@import '~carbon-components/scss/globals/scss/vars';
+
+div.simple-pagination-container {
```

Comment:
do we have to use these element selectors?

Reply:
Nope, changed them over to regular classnames